nolearn/lasagne/tests/test_base.py

```diff
@@ -114,6 +114,22 @@ def test_save_params_to_path(self, net, net_fitted, X_test, y_pred):
         net_loaded.load_params_from(path)
         assert np.array_equal(net_loaded.predict(X_test), y_pred)
 
+    def test_load_params_from_message(self, net, net_fitted, X_test, y_pred,
+                                      capsys):
+        net_loaded = clone(net)
+        net_loaded.verbose = 1
+        net_loaded.load_params_from(net_fitted)
+
+        out = capsys.readouterr()[0]
+        message = """Loaded parameters to layer 'hidden1' (shape 784x512).
+Loaded parameters to layer 'hidden1' (shape 512).
+Loaded parameters to layer 'hidden2' (shape 512x512).
+Loaded parameters to layer 'hidden2' (shape 512).
+Loaded parameters to layer 'output' (shape 512x10).
+Loaded parameters to layer 'output' (shape 10).
```

Comment:
Each layer appears twice, once for weights and once for bias. You can tell from the shape.